pkg/kubesphere/manifests.go

```diff
@@ -416,11 +416,11 @@ spec:
 )
 
 func GenerateKubeSphereYaml(repo, version string) (string, error) {
-	if os.Getenv("KKZONE") == "cn" {
+	if strings.Contains(version, "latest") && os.Getenv("KKZONE") == "cn" {
```

Comment:
How about these versions like v3.0.0, v3.1.0? Actually, I would suggest we sync all images into this registry which include the nightly build image and kubespheredev

Reply:
v3.0.0 does not support changing the namespace of images yet. Currently only the latest images are synced, this policy will be adjusted after other images are synchronized.

Reply:
Got you.